galadriel/tools/web3/onchain/solana/jupiter.py

```diff
@@ -44,7 +47,7 @@ class SwapTokenTool(SolanaBaseTool):
     """
 
     name = "jupiter_swap_token"
-    description = "Swaps one token for another on Jupiter Swap."
+    description = "Swaps one token for another on Jupiter Swap. It also supports Meteora, Fluxbeam, 1Intro, Pump.Fun, Raydium and Whirlpools."
```

Comment:
I don't think it does support all these things.

Reply:
It's stated by their official doc, haven't tried all these though https://station.jup.ag/guides/general/get-your-token-on-jupiter#how-to-get-your-pool-routed-on-jupiter